server/commands.lua

```diff
@@ -12,6 +12,7 @@ lib.addCommand('tp', {
     },
     restricted = 'group.admin'
 }, function(source, args)
+    if not IsOptin(source) then return Notify(source, locale('error.not_optin'), 'error') end
```

Comment:
Put an empty line under the returns for readability. Do this for all of the commands.

Reply:
Still unresolved, https://github.com/Qbox-project/qbx_core/pull/689/commits/16e8c9278703fc7844b13e0776e023c51c582541 doesn't address Berkie's feedback; this is how it should look like to improve readability:



```suggestion
}, function(source, args)
    if not IsOptin(source) then Notify(source, locale('error.not_optin'), 'error') return end

    if args[locale('command.tp.params.x.name')] and not args[locale('command.tp.params.y.name')] and not args[locale('command.tp.params.z.name')] then
```